packages/@react-spectrum/searchwithin/stories/SearchWithin.stories.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import {action} from '@storybook/addon-actions';
+import {Item, Picker} from '@react-spectrum/picker';
+import React from 'react';
+import {SearchField} from '@react-spectrum/searchfield';
+import {SearchFieldProps} from '@react-types/searchfield';
+import {SearchWithin} from '../';
+import {SpectrumPickerProps} from '@react-types/select';
+import {SpectrumSearchWithinProps} from '@react-types/searchwithin';
+
+export default {
+  title: 'SearchWithin'
+};
+
+function render(props: Omit<SpectrumSearchWithinProps, 'children'> = {}, searchFieldProps: SearchFieldProps = {}, pickerProps: Omit<SpectrumPickerProps<object>, 'children'> = {}) {
+  return (
+    <SearchWithin label="Search" {...props}>
+      <SearchField placeholder="Search" {...searchFieldProps} onChange={action('change')} onSubmit={action('submit')} />
+      <Picker defaultSelectedKey="all" {...pickerProps} onSelectionChange={action('selectionChange')}>
+        <Item key="all">All</Item>
+        <Item key="campaigns">Campaigns</Item>
+        <Item key="audiences">Audiences</Item>
+        <Item key="tags">Tags</Item>
+      </Picker>
+    </SearchWithin>
+  );
+}
+
+function renderReverse(props: Omit<SpectrumSearchWithinProps, 'children'> = {}, searchFieldProps: SearchFieldProps = {}, pickerProps: Omit<SpectrumPickerProps<object>, 'children'> = {}) {
+  return (
+    <SearchWithin label="Search" {...props}>
+      <Picker defaultSelectedKey="all" {...pickerProps} onSelectionChange={action('selectionChange')}>
+        <Item key="all">All</Item>
+        <Item key="campaigns">Campaigns</Item>
+        <Item key="audiences">Audiences</Item>
+        <Item key="tags">Tags</Item>
+      </Picker>
+      <SearchField placeholder="Search" {...searchFieldProps} {...searchFieldProps} onChange={action('change')} onSubmit={action('submit')} />
+    </SearchWithin>
+  );
+}
+
```

Comment:
I think we should have value, isQuiet and isReadOnly stories. Speaking of value, we should have a picker defaultValue story as well as searchwithin value or default value. It looks like your render has the option to set searchfieldprops and pickerprops so I'm guessing more interesting stories are planned.

Reply:
Added isReadyOnly, as well as default values for both the SearchField and Picker. This component won't support isQuiet, so no need for that. I'll get value added as well.